tools/bug-report/pkg/content/content.go

```diff
@@ -0,0 +1,184 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package content
+
+import (
+	"fmt"
+	"strings"
+
+	"istio.io/istio/tools/bug-report/pkg/kubectlcmd"
+	"istio.io/pkg/log"
+)
+
+const (
+	coredumpDir = "/var/lib/istio"
+)
+
+var (
+	istiodURLs = []string{
+		"debug/edsz",
```

Comment:
this and config_dump don't actually work without query params for the specific proxy so I don;t see how this can work

Reply:
i had it in there as a todo but removed it, will put back in future pr.